TestON/drivers/common/cli/onosclidriver.py

```diff
@@ -147,12 +147,12 @@ def setCell( self, cellname ):
                 # Expect the cellname in the ONOSCELL variable.
                 # Note that this variable name is subject to change
                 #   and that this driver will have to change accordingly
-                self.handle.expect( "ONOS_CELL" )
+                self.handle.expect(str(cellname))
```

Comment:
I know i told you to go with this one, but thinking about it some more, I think we should go with "ONOS_CELL". 

The reason for this is the cell name will always be there since we just sent that command and the command string will be in the buffer. ONOS_CELL is a part of the output of the command